src/mol-io/reader/lammps/schema.ts

```diff
@@ -0,0 +1,206 @@
+/**
+ * Copyright (c) 2017 mol* contributors, licensed under MIT, See LICENSE file for more info.
+ *
+ * @author Ludovic Autin <ludovic.autin@gmail.com>
+ */
+import { Column } from '../../../mol-data/db';
+
+interface LammpsUnitStyle {
+    mass: string;
+    distance: string;
+    time: string;
+    energy: string;
+    velocity: string;
+    force: string;
+    torque: string;
+    temperature: string;
+    pressure: string;
+    viscosity: string;
+    charge: string;
+    dipole?: string;
+    electricField?: string;
+    density: string;
+    scale: number;
+}
+
+export const unitStyles: { [key: string]: LammpsUnitStyle } = {
+    lj: {
```

Comment:
can we actually support that? How would that data look?

Reply:
I feel like its more like metadata that we can use to annotate/labels thing. I don't mind removing it.

Reply:
It's fine to keep it, just curious. Also, you can get the type and name for the param from this. Something like `keyof type unitStyles` and `Object.key(unitStyles)`. We usually uppercase the first letter for such constant data.